assets/training/finetune_acft_hf_nlp/components_v2/pipeline_components/chat_completion/spec.yaml

```diff
@@ -1,6 +1,6 @@
 $schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
 name: oss_chat_completion_pipeline
-version: 0.0.14
+version: 0.0.15
```

Comment:
Bump command components version too for this pipeline

Reply:
Thanks Aman, done.